app/src/main/java/org/mozilla/tv/firefox/webrender/WebRenderFragment.kt

```diff
@@ -154,7 +154,7 @@ class WebRenderFragment : EngineViewLifecycleFragment(), Session.Observer {
     }
 
     override fun onEngineViewCreated(engineView: EngineView): Disposable? {
-        return serviceLocator?.screenController?.currentActiveScreen?.subscribe {
+        return serviceLocator?.screenController?.currentActiveScreen?.distinctUntilChanged()?.subscribe {
```

Comment:
By convention, individual Rx operators are generally put on separate lines.  So this would be 

```kotlin
serviceLocator?.screenController?.currentActiveScreen
    ?.distinctUntilChanged()
    ?.subscribe {
```
or
```kotlin
serviceLocator?.screenController?.currentActiveScreen?.distinctUntilChanged()
    ?.subscribe {
```